src/componentTypesView.ts

```diff
@@ -173,10 +173,11 @@ export class ComponentTypesView implements TreeDataProvider<ComponentType> {
     }
 
     @vsCommand('openshift.componentTypesView.registry.add')
-    public static async addRegistryCmd(): Promise<void> {
+    public static async addRegistryCmd(registryContext: Registry): Promise<void> {
         // ask for registry
         const regName = await window.showInputBox({
-            prompt: 'Provide registry name to display in the view',
+            value: registryContext?.Name,
+            prompt: registryContext ? 'Edit registry name' : 'Provide registry name to display in the view',
```

Comment:
If one wants to change URL, there is no way to get to URL input unless name of the registry changed first.

Reply:
Yes, Since we named the option as "Edit", then user can edit the name/URL. So Only I have provided the placeholder value by default its existing name. Suppose if user don't want to edit the name then he can go the existing name by pressing enter. Otherwise we should provide option to user as edit name/edit URL, based on his input we can proceed. May I know your view about the same?

Reply:
I am fine with current implementation. Pressing enter does not work for me, I see error like this below
![image](https://user-images.githubusercontent.com/620330/168966128-fbd2bb61-e20e-408a-82dd-11255cb70497.png)
so I cannot get to URL input.